addons/mail/static/src/js/activity.js

```diff
@@ -494,8 +590,12 @@ var Activity = BasicActivity.extend({
                 nbOverdueActivities: nbActivities.overdue,
                 dateFormat: time.getLangDateFormat(),
                 datetimeFormat: time.getLangDatetimeFormat(),
+                widget: this,
```

Comment:
don't forget to remove `widget` if you do not use it in your template

Reply:
Well, in the end you still need it, because `HiddenInputFile` makes use of variable `widget`:
https://github.com/odoo/odoo/blob/master/addons/web/static/src/xml/base.xml#L1001

_(no linter and no documentation of qweb templates makes it hard to catch this kind of issues)_